NuGet.Clients.sln

```diff
@@ -121,6 +121,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.SolutionRestoreManage
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.SolutionRestoreManager.Test", "test\NuGet.Clients.Tests\NuGet.SolutionRestoreManager.Test\NuGet.SolutionRestoreManager.Test.csproj", "{F7837EEB-1B49-482F-8722-EB35BA0937CC}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.VisualStudio.Workspace.Extensions.Implementation", "src\NuGet.Clients\NuGet.VisualStudio.Workspace.Extensions.Implementation\NuGet.VisualStudio.Workspace.Extensions.Implementation.csproj", "{CB2D257A-7C56-4B2D-ACC8-BD9114088BC7}"
```

Comment:
This is NuGet specific extender code written by DPL folks. I've just got the ownership of this code, and included in our source code. I don't think we should merge this code into any of the existing project since this is specific to DPL and will only be loaded if DPL mode is enabled. Also I don't see any good place in current projects to include this piece.